src/integrations/intercom/services/conversations-getter.js

```diff
@@ -77,7 +77,7 @@ function ConversationsGetter(Implementation, params, opts, collectionName) {
 
             return P
               .map(conversations, (conversation) => {
-                if (conversation.assignee.type === 'admin') {
+                if (conversation.assignee !== null && conversation.assignee.type === 'admin') {
```

Comment:
```suggestion
                if (conversation.assignee && conversation.assignee.type === 'admin') {
```

We could be vague and consider that `conversation.assignee` should not be falsy.